eZ/Publish/SPI/Persistence/Content/Handler.php

```diff
@@ -269,6 +269,19 @@ public function removeRelation($relationId, $type);
      */
     public function loadRelations($sourceContentId, $sourceContentVersionNo = null, $type = null);
 
+    /**
+     * Counts relations from $destinationContentId only against published versions. Optionally, count only those with $type.
+     *
+     * @param int $destinationContentId Destination Content ID
+     * @param int|null $type {@see \eZ\Publish\API\Repository\Values\Content\Relation::COMMON,
+     *                 \eZ\Publish\API\Repository\Values\Content\Relation::EMBED,
+     *                 \eZ\Publish\API\Repository\Values\Content\Relation::LINK,
+     *                 \eZ\Publish\API\Repository\Values\Content\Relation::FIELD}
```

Comment:
`\eZ\Publish\API\Repository\Values\Content\Relation::ASSET` is missing on this list but IMHO the list is not needed (who will remember to update it when we introduce new relation type?).